lms/djangoapps/grades/admin.py

```diff
@@ -0,0 +1,25 @@
+"""
+Django admin page for grades models
+"""
+from django.contrib import admin
+
+from config_models.admin import ConfigurationModelAdmin
+
+from lms.djangoapps.grades.models import CoursePersistentGradesFlag, PersistentGradesEnabledFlag
+from lms.djangoapps.grades.forms import CoursePersistentGradesAdminForm
+
+
+class CourseAuthorizationAdmin(admin.ModelAdmin):
+    """Admin for enabling subsection grades on a course-by-course basis."""
+    form = CoursePersistentGradesAdminForm
+    fieldsets = (
```

Comment:
Can we also add a search_field for the course_id.

https://docs.djangoproject.com/en/1.10/ref/contrib/admin/#django.contrib.admin.ModelAdmin.search_fields